src/test/java/com/easypost/BetaRateTest.java

```diff
@@ -34,7 +73,11 @@ public static void setup() throws EasyPostException {
      */
     @Test
     public void testRetrieveStatelessRates() throws EasyPostException {
-        vcr.setUpTest("retrieve_stateless_rates");
+        List<MockRequest> mockRequests = new ArrayList<>();
+        mockRequests.add(new MockRequest(new MockRequestMatchRules(Requestor.RequestMethod.POST, ".*\\/rates.*"),
+                new MockResponse(200, retrieveRatesResponseJson)));
+
+        vcr.setUpTest("retrieve_stateless_rates", mockRequests);
```

Comment:
Question: with the mocking, it still makes the real API request and records to a cassette but is using the mock data instead of the API response?

Reply:
No, any mock request is not actually executed against the API.

Reply:
If this request is mocked which is not hitting the API endpoint, why do we need `vcr.setUpTest` and can we remove the existing corresponding cassettes (if there is one)?

Reply:
Sure. Wanted to get some feedback on this anyway, since we could keep the VCR structure and pass in mock requests to it (allowing us to do a combination of mock and real requests), or remove the VCR and use a mock client instead (allowing only mock requests). For tests that only make one API call, it can go either way.

Thoughts?

Reply:
We probably don't need both ways. I would remove the VCR and use a mock client, this seems the easier way for any endpoints that we need to mock

Reply:
Mocking, (as we've previously discussed at length), should be used as a last resort. It doesn't execute real code, it doesn't make real API calls, and it's very prone to becoming out of date or making things messy (ala ~30+ lines above of concatenated strings.

We should be making real API calls and using VCR unless it's out of the question (eg: billing tests)

Reply:
Options:

1. change the assertions (we should have been asserting that the rate field is a float instead of an exact value) - this problem should also get better after July 9 once those USPS rates adjust (the service name specifically). May honestly be more concrete
2. only mock the data we absolutely need in its most simple form

Reply:
Correct. This attempts to do Option 2 (and will be cleaned up)